bessctl/conf/samples/exactmatch.bess

```diff
@@ -33,12 +33,12 @@ em:1 -> Sink()
 em:2 -> Sink()
 em:5 -> Sink()   # used as default gate
 
-em.add(fields=[chr(17), aton('172.16.100.1'), aton('10.0.0.1')], gate=0)
-em.add(fields=[chr(6), aton('172.12.55.99'), aton('12.34.56.78')], gate=1)
-em.add(fields=[chr(17), aton('172.12.55.99'), aton('12.34.56.78')], gate=2)
+em.add(fields=[{'value_int':17}, {'value_bin':aton('172.16.100.1')}, {'value_bin':aton('10.0.0.1')}], gate=0)
```

Comment:
I was curious and was looking at how to implement such a thing (it's still not quite clear, I think you can inspect for the `.oneofs` or `.oneofs_by_name` field to figure them out) but it wonder if it might also be a bit error prone...